Sources/Kommunicate/Classes/Kommunicate.swift

```diff
@@ -736,6 +795,38 @@ open class Kommunicate: NSObject, Localizable {
             }
         }
     }
+    
+    class func openChatOnView(
```

Comment:
Change the name to `openChatIn` and keep rootView as first param.

Reply:
done